avr-specs/sync-from-upstream.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
```

Comment:
The recommended shebang is `/usr/bin/env python3` because that composes better with virtualenv and the likes:

```suggestion
#!/usr/bin/env python3
```